build.gradle

```diff
@@ -12,6 +12,7 @@ allprojects {
     repositories {
         jcenter()
         maven { url "https://jitpack.io" }
+        mavenLocal()
```

Comment:
do we need mavenLocal()? Could it create a  "it works on my computer but not online" error?

Reply:
No we should remove it, I will remove once Ive updated the references to the released dependencies.